packages/components/src/toggle-group-control/toggle-group-control-option/component.tsx

```diff
@@ -20,13 +20,28 @@ import {
 	useContextSystem,
 	WordPressComponentProps,
 } from '../../ui/context';
-import type { ToggleGroupControlOptionProps } from '../types';
+import type { ToggleGroupControlOptionProps, WithToolTipProps } from '../types';
 import { useToggleGroupControlContext } from '../context';
 import * as styles from './styles';
 import { useCx } from '../../utils/hooks';
+import Tooltip from '../../tooltip';
 
 const { ButtonContentView, LabelPlaceholderView, LabelView } = styles;
 
+const WithToolTip = ( {
+	text,
+	children,
+}: WordPressComponentProps< WithToolTipProps, 'div' > ) => {
```

Comment:
I don't believe there's a need to use `WordPressComponentProps` here

```suggestion
}: WithToolTipProps ) => {
```